### DISCUSSION TITLE
License Purchase Questions - Figma UI KIt

### DISCUSSION THREAD

**Discussion by abigaili99** [primefaces/community#2481] [Figma UI Kit]
Hello, we have 2 designers on our team and many developers. Could you please help me understand which license we need for the Figma UI kit based on the information below? 

- 2 designers need edit access to the UI kit file (customize the colors to match our brand and create custom components) 
- Developers need to have "dev mode" access on the UI kit file to check out specifications etc
- Once customizations are done, the designers would like to publish the UI kit file as a Figma library so that others in the team can use the brand-styled components in their own Figma files (they do not need edit access to the UI kit file)

Also, how does the licensing work? If we purchase the "team" license, do we get 5 slots for edit access users on the one file or 5 files? 
Another question, what is the cost for another year of updates? This is not clear on your website, and I need an idea of the total costs per year before I can purchase.

Thank you!

**Comment by onursenture** [comment]
Hello,

When choosing a PrimeOne UI Kit license tier, the only important factor is how many designers will use it. It doesn't matter how many files you create with the UI Kit or how many developers use dev mode to inspect the files or the UI Kit itself.

The upgrade fee is $49 for the Single Designer tier and $149 for the Team tier, but these prices may change.

Sorry for the late response. I hope this helps.
